api/client/webclient/webclient.go

```diff
@@ -46,6 +47,9 @@ func newWebClient(insecure bool, pool *x509.CertPool) *http.Client {
 				RootCAs:            pool,
 				InsecureSkipVerify: insecure,
 			},
+			Proxy: func(req *http.Request) (*url.URL, error) {
+				return httpproxy.FromEnvironment().ProxyFunc()(req.URL)
+			},
```

Comment:
```suggestion
			Proxy: httpproxy.FromEnvironment().ProxyFunc(),
```

Reply:
@codingllama Can you take one more look at this when you have a moment?

Reply:
This would be nice, but the function returned by `ProxyFunc()` expects a URL while `Proxy:` expects it to be a Request.